dotCMS/src/main/java/com/dotmarketing/filters/CMSFilter.java

```diff
@@ -172,11 +173,16 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
             return;
         }
 
-        if (iAm == IAm.PAGE) {
+        else if (iAm == IAm.PAGE) {
             // Serving a page through the velocity servlet
             StringWriter forward = new StringWriter();
-            forward.append("/servlets/VelocityServlet");
-
+            if(PageMode.get(request).showLive) {
+                forward.append("/servlets/VelocityLiveServlet");
+            }
+            else {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "else" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)